debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerChangeTablePointer.java

```diff
@@ -47,16 +47,22 @@
     private ResultSetMapper<Object[]> resultSetMapper;
     private final ResultSet resultSet;
     private final int columnDataOffset;
+    private final SourceTimestampMode sourceTimestampMode;
 
-    public SqlServerChangeTablePointer(SqlServerChangeTable changeTable, ResultSet resultSet) {
-        this(changeTable, resultSet, COL_DATA);
+    public SqlServerChangeTablePointer(SqlServerChangeTable changeTable, ResultSet resultSet, SourceTimestampMode sourceTimestampMode) {
+        this(changeTable, resultSet, sourceTimestampMode, COL_DATA);
     }
 
-    public SqlServerChangeTablePointer(SqlServerChangeTable changeTable, ResultSet resultSet, int columnDataOffset) {
+    public SqlServerChangeTablePointer(SqlServerChangeTable changeTable, ResultSet resultSet, SourceTimestampMode sourceTimestampMode, int columnDataOffset) {
```

Comment:
Looks like this could be private? In other words, do we even need the two constructors?

Reply:
One constructor is enough.